src/screens/MemberDetail/MemberDetail.tsx

```
@@ -130,15 +143,23 @@ const MemberDetail: React.FC<MemberDetailProps> = ({ id }): JSX.Element => {
     }
   };
 
+  // to handle deletion of user profile
+  const handleDeleteUser = (): void => {
+    navigate('/');
+  }
+
   // to handle the change in the form fields
-  const handleFieldChange = (fieldName: string, value: string): void => {
+  const handleFieldChange = (fieldName: string, value: string | Boolean): void => {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)